srcDoc/command_line_parameters.rst

```diff
@@ -422,6 +422,22 @@ Test Parameters for Publishing and Subscribing Applications
    **Default:**
    ``Not set. The priority will not be modified.``
 
+-  ``-cacheStats``
+
+   Enable extra messages showing the reader/writer queue sample count and
+   sample count peak.
+
+   In the publisher side also shows the *Pulled Sample Count* of the Writer.
+
+   The frequency of these log messages will be determined by the
+   ``-latencyCount`` in the Publisher side, since the message is only shown
+   after a *latency ping*. In the subscriber side it will be shown after once
+   every second.
+
+   This option is just available for the *C++ Traditional* API implementation.
+
+   **Default:** ``Not enabled``
```

Comment:
Why we do not want this feature enabled as default?

Reply:
It adds another line of data, and makes the output more difficult to read, also, it adds calls to the critical path.